tests/lib/test-utils.js

```diff
@@ -13,7 +13,9 @@ module.exports = {
     if (global.location) {
       return urlUtils.getOrigin(global.location.href);
     }
-    return 'http://localhost:8081';
+    return /^v0\.(?:8|10)/.test(process.version)
```

Comment:
Well it makes sense, it all depends on what are the supported versions, but I wouldn't add `0.6`, it's really obsolete.